package.json

```diff
@@ -36,6 +36,11 @@
 		"chromium_revision": "662092",
 		"chromium_version": "76.0.3803.0"
 	},
+	"husky": {
```

Comment:
Since now we are using `husky` to manage hooks, can you update the `pre-commit` hook as well?
Then we can remove the `pre-commit` dependency as well.

Reply:
Done!